templates/taxi/car_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list<a style="float: right" href="{% url "taxi:car-create" %}">+</a></h1>
```

Comment:
Consider adding a button element for the '+' link for better accessibility. Screen readers and assistive technologies can better interact with <button> elements.

Reply:
done